core/src/consensus/validator_set/validator_list.rs

```diff
@@ -86,6 +86,17 @@ impl ValidatorSet for RoundRobinValidator {
         self.validators.len()
     }
 
+    fn normalized_voting_power(
+        &self,
+        parent: &BlockHash,
+        _signer_idx: usize,
```

Comment:
This function is in the trait `ValidatorSet'. When `DynamicValidator` is activated, the normalized voting power could only be fetched using signer index to look up its total delegation. However, `RoundRobinValidator` don't have to use the index value because the normalized voting power is determined by dividing the total power equally.